src/thorlabs_cube/driver/tcube/tpz.py

```diff
@@ -115,30 +114,30 @@ async def set_output_position(self, position_sw):
             [0; 65535] depending on the unit. This corresponds to 0 to 100% of
             the maximum piezo extension.
         """
-        payload = st.pack("<HH", 1, position_sw)
+        payload = st.pack("<HH", _CHANNEL, position_sw)
```

Comment:
```suggestion
        payload = st.pack("<HH", Tpz._CHANNEL, position_sw)
```
Use class constants.